packages/server/src/tests/jestSetup.ts

```diff
@@ -2,17 +2,11 @@ import env from "../environment"
 import { env as coreEnv, timers } from "@budibase/backend-core"
 import { testContainerUtils } from "@budibase/backend-core/tests"
 
-if (!process.env.DEBUG) {
-  global.console.log = jest.fn() // console.log are ignored in tests
-  global.console.warn = jest.fn() // console.warn are ignored in tests
-}
```

Comment:
Removing this didn't really seem to make a difference, and I've been confused in the past about where my `console.log` statements went. Because we lint against this now, I don't think we need it.